app/api/health.py

```
@@ -0,0 +1,337 @@
+
+            response_time = (time.time() - start_time) * 1000
+
+            return {
+                "status": "healthy",
+                "response_time_ms": round(response_time, 2),
+                "version": info.get("redis_version"),
+                "uptime_seconds": info.get("uptime_in_seconds"),
+                "connected_clients": info.get("connected_clients", 0),
+                "used_memory": info.get("used_memory_human"),
+            }
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)